yb-voyager/cmd/live_migration.go

```diff
@@ -135,14 +140,103 @@ func streamChanges(state *ImportDataState, tableNames []sqlname.NameTuple) error
 		}
 		log.Infof("got next segment to stream: %v", segment)
 
-		err = streamChangesFromSegment(segment, evChans, processingDoneChans, eventChannelsMetaInfo, statsReporter, state)
+		err = streamChangesFromSegment(segment, evChans, processingDoneChans, eventChannelsMetaInfo, statsReporter, state, tableToPartitioningStrategyMap)
 		if err != nil {
 			return fmt.Errorf("error streaming changes for segment %s: %v", segment.FilePath, err)
 		}
 	}
 	return nil
 }
 
+const (
+	PARTITION_BY_PK    = "pk"
+	PARTITION_BY_TABLE = "table"
+)
+
+func handleCdcPartitioningStrategy(tableNames []sqlname.NameTuple) (*utils.StructMap[sqlname.NameTuple, string], error) {
```

Comment:
This function is doing two things: 
1. fetching the cdc partitioning strategy ffrom metadb
2. figuring it out (first time) the cdc partitioning strategy per table, adn then saving it in metadb. 

Can we add documentation above this function?
And also perhaps name it something like : `GetCdcPartitioningStrategyPerTable`